src/main/scala/io/iohk/ethereum/utils/Config.scala

```diff
@@ -22,6 +22,8 @@ object Config {
 
   val keysFile: String = config.getString("keys-file")
 
+  val shutdownTimeout: Duration = config.getDuration("shutdown-timeout").toMillis.millis
```

Comment:
Is this added just for purity or do you think it might for example prevent those NPEs that we sometimes see when  closing the app?